gm4_podzol_rooting_soil/data/gm4_podzol_rooting_soil/tags/blocks/air.json

```diff
@@ -1,16 +1,8 @@
 {
   "values":[
-    "minecraft:air",
-    "minecraft:cave_air",
-    "minecraft:void_air",
-    "minecraft:grass",
+    "#gm4:replaceable",
     "#minecraft:flowers",
```

Comment:
If my comment in the #gm4:foliage thing about mushrooms is added, this can be changed to #gm4:foliage